### ISSUE TITLE
Can't sign transactions with Ledger Nano S through Base Aepp

### ISSUE THREAD

**Issue by ggeorgiev87** [aeternity/aepp-base#1427]
I am having an issue when using the Base Aepp where it connects to my Ledger Nano S, the amount of AE that I have is displayed correctly and everything works fine, but when I try to sign a transaction and send AE out of my wallet it says "Ledger is not connected" and does not prompt me on my ledger to sign the transaction. 
I saw that there are a fair number of users complaining about this on various forums. 
It basically means that I am unable to access my tokens because as far as I know, the Base Aepp is the only way of interacting with the Ledger. 
I tried turning on and off the ledger, clearing the cache of the browser and all the routine things, but nothing helped. 

![telegram-cloud-photo-size-4-5906591200901707215-y](https://user-images.githubusercontent.com/90618152/133077653-fa774267-6367-42a0-8c52-cbd66c035572.jpg)

**Comment by marc0olo** [comment]
@ggeorgiev87 can you check and confirm that you have installed the latest version of the aeternity application via Ledger Live? please provide us an information about the application version you have installed on the Ledger Nano S right now

**Comment by ggeorgiev87** [comment]
@marc0olo  yes I do have the latest version installed of both the application and Ledger Nano's software and Ledger Live application. 
Here is the info:
![telegram-cloud-photo-size-4-5906591200901707260-y](https://user-images.githubusercontent.com/90618152/133227389-130a6f62-95c0-477c-952d-adf1863fcb82.jpg)

**Comment by marc0olo** [comment]
now this is interesting. seems like I have definitely other versions installed:
![image](https://user-images.githubusercontent.com/8124114/133293043-9e58e151-6682-48ba-88a6-b17289ebf399.png)

@ggeorgiev87 how urgent is this? I am not sure if we can investigate shortly. I know there will come Ledger support to the Superhero Wallet. shouldn't be far away and hopefully introduced in the near future
